### PR TITLE
Added table loaders

### LINKED ISSUE #3198: NO content Loaders in dashboard section
**Expected Behaviour**

there should be content loaders in dashboard section too.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by anishagg17**
Fixes #3198

Changes: Used react-content-loader to implement loaders for tables

Demo Link: https://pr-3199-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 
Screenshot 2020-01-03 at 10 01 01 PM

### CODE REVIEW

File: src/components/cms/Dashboard/TableLoader.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```
Comment (akshatnitd) [inline src/components/cms/Dashboard/TableLoader.js]: Put this TableLoader in the shared folder and try to use it at all places where table is used.

Comment (anishagg17) [reply src/components/cms/Dashboard/TableLoader.js]: Yes Sir , I have already  used it in all places possible

**Comment by anishagg17** [comment]
> Also, each skeleton row has too many details. Just keep 4 dashes.

Done Sir!

**Comment by anishagg17** [comment]
> Each dash in a column should be of same length. I see variable length there.

Done Sir!

**Comment by anishagg17** [comment]
Preview:


Screenshot 2020-01-04 at 6 02 51 PM

**Pull request merged**